Source/Csla/DataPortalClient/DataPortalProxy.cs

```diff
@@ -44,7 +44,7 @@ public DataPortalProxy(ApplicationContext applicationContext)
     /// Gets or sets the Client timeout
     /// in milliseconds (0 uses default timeout).
     /// </summary>
-    public virtual int Timeout { get; set; }
+    public virtual int Timeout { get; }
```

Comment:
Could this have a protected set instead of delegating the field down to subclasses?

Reply:
Also, let's make this a TimeSpan

Reply:
Or we could remove this property and let just the options in the http proxy, even because that this timeout means different things in http and rabbitmq

Reply:
Excellent - let's make that happen.